src/main/java/com/imyvm/essential/systems/ptt/PlayTimeTrackSystem.java

```diff
@@ -41,7 +46,19 @@ public Text getPlayTimeText(PlayerTrackData trackData) {
             TrackData data = counter.getGetDataFunction().apply(trackData);
             Text name = tr("name.ptt.category." + counter.getTypeId());
             int duration = (int) (data.getDuration() / 1000);
-            Text status = tr("name.ptt.status." + data.getStatus().toString().toLowerCase());
+            String statusStr = "name.ptt.status." + data.getStatus().toString().toLowerCase();
+            Text status;
+            if (statusStr.equals("name.ptt.status.not_meet")){
+                switch (counter.getTypeId()) {
+                    case "day" -> status = tr(statusStr, TimeUtil.formatDuration(DurationConstants.DAY - duration));
+                    case "week" -> status = tr(statusStr, TimeUtil.formatDuration(DurationConstants.WEEK - duration));
+                    case "month" -> status = tr(statusStr, TimeUtil.formatDuration(DurationConstants.MONTH - duration));
+                    case "year" -> status = tr(statusStr, TimeUtil.formatDuration(DurationConstants.YEAR - duration));
+                    default -> status = tr("name.ptt.status.continuous");
```

Comment:
```suggestion
                status = tr(statusStr, TimeUtil.formatDuration(counter.getTimeRequired() - duration));
```

然后给 `TimeCounter` 加一个 getter `getTimeRequired()`

然后记得把 `DurationConstants` 删了